docs/pim/create_product.md

```diff
@@ -0,0 +1,63 @@
+---
+description: Create new products or modify existing ones.
+---
+
+# Create product
+
+If your [user role](../permission_management/work_with_permissions.md) has the `Product/Edit` permission, you can modify product and add individual attributes or attribute groups. You can create new [products](products.md#products) to present your offer in the website, including product specification, and pricing. 
+Each product has a unique identifying product code. 
+
+!!! note
+
+    To create a product you must first create [product type](products.md#product-types).
+
+## Manually create product
+
+You can create product in several ways:
+
+- In the left panel, go to **Product catalog** -> **Products** and click **Create** button.
+- In the left panel, go to **Content** -> **Content structure** -> **Product catalog** and click **Create content** button.
+- In the left panel, go to **Content** -> **Dashboard** and click **Create content** button.
+- In the search bar type 'product' -> click **Product catalog** and click **Create** button.
+
+## Add single product
+
+1\. In the left panel, go to **Product catalog** -> **Products** and click **Create** button. 
+
+![Adding a Product](img/add_product.png "Adding a Product")
+
+2\. Choose product type.
+
+3\. Click **Add** button.
+
+4\. Fill in basic product information: name, product specification, description and, if needed, other specific attributes. 
```

Comment:
Sidenote: product structure is customizable on product type level

Reply:
Ok, description updated. 
As we discussed - let's wait for the update of 'Content Items' article, then we can add the link.